src/utils.js

```diff
@@ -88,3 +90,36 @@ export function ignorePrivateFunctions(list) {
 
   return filteredList;
 }
+
+
+export function isLocalUrl(url_) {
```

Comment:
Hmm, I should probably use es6 modules import syntax to modify the url lib or just import parse and then I can use url here.